docs/architecture/cloud-native/cloud-native-data.md

```diff
@@ -1,9 +1,10 @@
 ---
```

Comment:
This is not a good file name per SEO guidelines. It repeats the words in the path.

Should simply be data.md

Reply:
Nish wants to address renaming in a separate POC.